libs/protocol-client/src/lib/protocol-types.ts

```diff
@@ -2188,6 +2187,7 @@ export type Contribution = {
   status: ContributionStatus;
   status_id: Scalars['Int'];
   tweet?: Maybe<TwitterTweet>;
+  tx_hash?: Maybe<Scalars['String']>;
```

Comment:
I think the only thing missing is to make sure this field is added to the permissions in the protocol-api.